recipes/mmcif-pdbx/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "mmcif-pdbx" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/a5/3f/73ca7acc16ac2f5cc99077a8b812168b2fb8ae99f4ee606183010c83b4ed/mmcif_pdbx-1.1.1.tar.gz"
```

Comment:
please look at the other python packages, it would be nice to use the `name` and `version` here as well. Should the name use a `_` instead of a `-`